code/planning/src/behavior_agent/behaviors/lane_change.py

```
@@ -87,10 +87,18 @@ def update(self):
-            self.change_distance = lane_change.distance
             self.change_detected = lane_change.isLaneChange
             self.change_option = lane_change.roadOption
             self.change_position = lane_change.position
+            # get change distance from global change point (transfered to local
+            #  hero coords) as this is more accurate than lanechange msg distance
+            hero_transform = _get_global_hero_transform()
+            local_pos: Point2 = (
+                hero_transform.inverse()
+                * Point2.new(self.change_position.x, self.change_position.y)
+            )
+            trajectory_local = mapping_common.mask.ros_path_to_line(trajectory_local)
```

Ruff (0.8.2)

Undefined name `trajectory_local`

(F821)